frontend/src/components/Accounts.js

```diff
@@ -0,0 +1,74 @@
+import { useState,useEffect } from "react"
+import Account from "./Account"
+import NewAccount from "./NewAccount"
+import Graph from "./Graph"
+import { useNavigate } from "react-router-dom"
+
+const Accounts=({currUser, setCurrUser, setTotAccts})=>{
+    const [accounts, setAccounts]=useState([])
+    const navigate = useNavigate();
+    const [updates, setUpdates]=useState(0)
+    const tota = [1]
+    accounts.map((account) => tota.push(account.amount))
+    setTotAccts(tota.reduce((sum, n) => sum + n))
```

Comment:
I am confused at what is happening here.  Part of that is the partial variables which I am unsure what they represent.  The second is that it look like you just want to find the amounts in the accounts.  The current way seems to be adding 1 to the value to that.  You can clean this up by using the second argument in the `reduce` function.  It would look like this:

```js
useEffect(() => {
  const totalOfAllAccounts = accounts.reduce((sum, account) => sum + account.amount, 0 /* the 0 here is the starting value */)

}, [accounts])
```

Reply:
not sure what I was doing here either and looking at it now it looks really janky--I guess that's a good sign I'm growing!